code/src/App.js

```diff
@@ -1,9 +1,16 @@
-import React from 'react'
+import React from 'react';
+
+import { ThoughtsList } from 'Components/ThoughtsList';
```

Comment:
components folder should not be written in uppercase, neither do created like that. Only component files should be written in uppercase